README.md

```diff
@@ -66,8 +63,8 @@ After cloning this repository, build with following command.
 mvn package 
 ```
 
-Running Mini Instance
----------------------
+Running via Maven
```

Comment:
I tried running via Maven.  A lot of logging is output.  Maybe, the README should be changed to tell people to redirect output and background the process.
```
mvn exec:java -Dexec.mainClass=phrasecount.cmd.Mini -Dexec.args="/tmp/mac fluo.properties" -Dexec.classpathScope=test &> mini.log & 
```